exploration/open-close-expressions.md

```diff
@@ -195,13 +201,40 @@ Markup is not valid in _declarations_ or _selectors_.
 
 * Introduces two new sigils, the pound sign `#` and the forward slash `/`.
 
-* As in HTML, differentiating "open" and "standalone" elements relies on registry information,
-  or in translations matching the structure used in the source
-  A rather clunky `{#foo/}` syntax was considered for explicitly-standalone elements,
-  but this did not reach consensus support.
-
 * In Mustache, the `{{#foo}}`...`{{/foo}}` syntax is used for *control flow* statements rather than printable data.
 
+### Standalone Markup
+
+_Standalone_ markup remains an open question.
+Three alternatives have been suggested:
+
+* Introduce additional syntax inspired by XML: `{#foo/}`.
+
+  This approach encodes the _standalone_ aspect in the data model, and doesn't rely on external sources of truth,
+  at the expense of introducing new syntax, which is admittedly a bit clunky.
+
+  It also creates a dichotomy between standalone _markup_ and regular _placeholders_, which can be argued to be _standalone_ as well.
+
+* Differentiate _open_ and _standalone_ elements based on the information stored in the registry,
+  or based on some other external source of truth
+  (e.g. the structure of the source to which the message is applied).
+
+  This approach offers a cleaner syntax at the expense of relying on external sources of truth for telling open and standalone elements apart.
+  Without the access to these sources of truth (e.g. a custom registry), standalone markup will be interpreted as open elements.
```

Comment:
Without access to some implementation, _all_ forms of markup are just so much noise.

I'm slightly sad that we didn't show using namespaces in the design document, since I suspect that `{#html:span id=foo}I am spanned{/html:span}` is friendlier to tools, given `{#ttml:span id=foo}I am timed text{/ttml:span}` exists too. Namespacing suggests how implementations would solve the problem of "external knowledge". There's no way that any of this works without some form of external information anyway.

Reply:
Yes, but you can get a lot of mileage without an implementation, just working with the data model, especially in tooling.

Reply:
I think namespaces are important for markup regimes to work. They won't be built-in to the default registry, so there needs to be a way to plug them in and for tools _and implementations_ to know what they mean.